RPRMakie/examples/volume.jl

```diff
@@ -1,16 +1,44 @@
-using RPRMakie
+using RPRMakie, ProgressMeter
 using Makie, NIfTI, FileIO
 using GLMakie
 r = LinRange(-1, 1, 100)
 cube = [(x .^ 2 + y .^ 2 + z .^ 2) for x = r, y = r, z = r]
 
+RPRMakie.activate!(
+    iterations = 32,
```

Comment:
32 seems to much, if this is suppose to be interactive for the user, as for the example maybe even more than 100 will be ok, since is not running in CI anyway. So, I would say iterations=1, and a comment to the user to increase that number if he wants a nice render.

Reply:
Fair enough.  5 should be fine in that case (without that, there's way too much noise in the render).
```suggestion
RPRMakie.activate!(
    iterations = 5, # increase this if you want a better render!
```